frontend/src/metabase/parameters/components/ParameterWidget.jsx

```diff
@@ -111,9 +111,15 @@ export default class ParameterWidget extends Component {
                 p => p.name === parameter.name && p.id !== parameter.id,
               ),
             })}
-            value={parameter.name}
-            onChange={e => setName(e.target.value)}
-            onBlur={() => this.setState({ isEditingName: false })}
+            value={this.state.editingNameValue}
```

Comment:
I try to always declare every state property when defining state, so just add `editingNameValue: null` in the ` state = {` block at the top.